frontend/public/extend/devconsole/components/ImportFlowForm/ImportFlowForm.tsx

```diff
@@ -547,6 +699,17 @@ export class ImportFlowForm extends React.Component<Props, State> {
             />
             <HelpBlock>{namespaceError}</HelpBlock>
           </FormGroup>
+          {this.state.showSourceSecretDropDown ? (
```

Comment:
No need to use ternary operator if you only care about truthy condition.
```suggestion
          {this.state.showSourceSecretDropDown && <SourceSecretSelector />
```